README.md

```diff
@@ -79,79 +145,66 @@ cp ~/YOUR/PATH/TO/mimTitles.txt ~/.local/share/wags_tails/omim/omim_<date>.tsv
 
 MetaKB relies on environment variables to set in order to work.
 
-* Always Required:
-  * `UTA_DB_URL`
-    * Used in Variation Normalizer which relies on UTA Tools
-    * Format: `driver://user:pass@host/database/schema`
-    * More info can be found [here](https://github.com/GenomicMedLab/uta-tools#connecting-to-the-database)
+- Always Required:
+  - `UTA_DB_URL`
+    - Used in Variation Normalizer which relies on UTA Tools
+    - Format: `driver://user:pass@host/database/schema`
+    - More info can be found [here](https://github.com/GenomicMedLab/uta-tools#connecting-to-the-database)
 
     Example:
 
     ```shell script
     export UTA_DB_URL=postgresql://uta_admin:password@localhost:5432/uta/uta_20210129
     ```
 
-* Required when using the `--load_normalizers_db` or `--force_load_normalizers_db` arguments in CLI commands
-  * `UMLS_API_KEY`
-    * Used in Therapy Normalizer to retrieve RxNorm data
-    * RxNorm requires a UMLS license, which you can register for one [here](https://www.nlm.nih.gov/research/umls/index.html). You must set the `UMLS_API_KEY` environment variable to your API key. This can be found in the [UTS 'My Profile' area](https://uts.nlm.nih.gov/uts/profile) after singing in.
+- Required when using the `--load_normalizers_db` or `--force_load_normalizers_db` arguments in CLI commands
+  - `UMLS_API_KEY`
+    - Used in Therapy Normalizer to retrieve RxNorm data
+    - RxNorm requires a UMLS license, which you can register for one [here](https://www.nlm.nih.gov/research/umls/index.html). You must set the `UMLS_API_KEY` environment variable to your API key. This can be found in the [UTS 'My Profile' area](https://uts.nlm.nih.gov/uts/profile) after singing in.
 
     Example:
 
     ```shell script
     export UMLS_API_KEY={rxnorm_api_key}
     ```
 
-  * `HARVARD_DATAVERSE_API_KEY`
-    * Used in Therapy Normalizer to retrieve HemOnc data
-    * HemOnc.org data requires a Harvard Dataverse API key. After creating a user account on the Harvard Dataverse website, you can follow [these instructions](https://guides.dataverse.org/en/latest/user/account.html) to generate a key. You will create or login to your account at [this](https://dataverse.harvard.edu/) site. You must set the `HARVARD_DATAVERSE_API_KEY` environment variable to your API key.
+  - `HARVARD_DATAVERSE_API_KEY`
+    - Used in Therapy Normalizer to retrieve HemOnc data
+    - HemOnc.org data requires a Harvard Dataverse API key. After creating a user account on the Harvard Dataverse website, you can follow [these instructions](https://guides.dataverse.org/en/latest/user/account.html) to generate a key. You will create or login to your account at [this](https://dataverse.harvard.edu/) site. You must set the `HARVARD_DATAVERSE_API_KEY` environment variable to your API key.
```

Comment:
I think my markdownlint reformatted all of this stuff - I can change it back if needed just lmk

Reply:
tbh, all of this can probably just be removed, it makes more sense to just let it live in the docs https://github.com/cancervariants/metakb/blob/main/docs/source/install.rst#gene-disease-and-therapy-normalizers

Reply:
yeah ngl, I did not do this part of the setup

Reply:
also dangit, forgot to update the docs